src/components/planner/sidebar/sessionController/course-picker/MajorSearchCombobox.tsx

```diff
@@ -48,7 +45,7 @@ const MajorSearchCombobox = ({ selectedMajor, setSelectedMajor }: Props) => {
           aria-expanded={open}
           className="w-full justify-between dark:bg-darker dark:text-slate-50"
         >
-          {selectedMajor ? majors.find((major) => major.id === selectedMajor.id)?.name : 'Seleciona um curso...'}
+          {selectedMajor ? selectedMajor.name : 'Seleciona um curso...'}
```

Comment:
In this line do you think that we should also display the name of the faculty along the `selectedMajor.name` as it now also appears on the search results?

Reply:
No need, since it also doesn't display the acronym. But if the team decides it could improve the experience, we can add it later